templates/mdnext-blog/pages/_app.js

```diff
@@ -1,6 +1,16 @@
 import { Chakra } from '@components/Chakra';
 import { DefaultSeo } from 'next-seo';
+import getShareImage from '@jlengstorf/get-share-image';
 
+const socialImage = getShareImage({
+  title: 'Nextjs + MDX Starter pack',
+  tagline: 'A test of sorts.',
+  cloudName: process.env.CLOUDINARY_USER_ID,
```

Comment:
could we change this to CLOUDINARY_CLOUDNAME

We will utilize Cloudinary throughout a few spots in `mdnext` so it'll be dope to centralize our env variables